build/cross-images/x86_64-unknown-linux-musl.dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ekidd/rust-musl-builder:latest
+FROM evanxg852000/cross-x86_64-unknown-linux-musl:latest
```

Comment:
seems that we need to remove `evanxg852000`? Do we?

Reply:
So the original image has not been updated to last rust version. this triggers a build error for `edition 2021`. 
I created an issue on the repo to update but it seems he is not been interested. In the first place, I made a script to build the image alongside but my fear is that they could just update the repo one day and our build will start failing. So I forked the repo and pushed the image on my dockerhub. Until he publishes a new version of the image.
The is nothing wrong with his repo from my check he seems just not to have time anymore to update the image.
https://github.com/emk/rust-musl-builder/issues/130

Reply:
Can we push this image somewhere on quickwit's dockerhub then? (and it's dockerfile on our github)

Reply:
Ok we have two images like this, I will move all of them into Quickwit repos then

Reply:
Yes forking sounds good to me.